app/src/androidTest/kotlin/net/rafaeltoledo/social/setup/TestSetup.kt

```diff
@@ -2,15 +2,13 @@ package net.rafaeltoledo.social.setup
 
 import android.app.Application
 import android.content.Context
-import android.support.test.InstrumentationRegistry
-import android.support.test.runner.AndroidJUnitRunner
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.runner.AndroidJUnitRunner
 import net.rafaeltoledo.social.TestSocialApp
-import org.junit.After
-import org.koin.standalone.StandAloneContext
 
 abstract class BaseInstrumentedTest {
 
-    protected val app: TestSocialApp by lazy { InstrumentationRegistry.getTargetContext().applicationContext as TestSocialApp }
+    protected val app: TestSocialApp by lazy { InstrumentationRegistry.getInstrumentation().targetContext.applicationContext as TestSocialApp }
```

Comment:
```suggestion
    protected val app: TestSocialApp by lazy { InstrumentationRegistry.instrumentation.targetContext.applicationContext as TestSocialApp }
```

Just to keep style consistent

Reply:
Seems that [ApplicationProvider](https://developer.android.com/reference/androidx/test/core/app/ApplicationProvider) is the best way to obtain an application instance now